core/src/Kokkos_View.hpp

```diff
@@ -39,6 +39,7 @@ static_assert(false,
 #ifdef KOKKOS_ENABLE_IMPL_MDSPAN
 #include <View/MDSpan/Kokkos_MDSpan_Extents.hpp>
 #include <View/MDSpan/Kokkos_MDSpan_Layout.hpp>
+#include <View/MDSpan/Kokkos_MDSpan_Accessor.hpp>
```

Comment:
Any good reason not to include in alphabetic order?

Reply:
They are in template argument order (Extents, Layout, Accessor).